### ISSUE TITLE
Wrong application name on Gnome

### ISSUE THREAD

**Issue by Dansoftowner** [Document-Archiver/com.sophisticatedapps.archiving.document-archiver#14]
Hi @stephansann ,

On Ubuntu a wrong application name is displayed on the top bar:
![screenshot](https://user-images.githubusercontent.com/49782833/117347180-0ade9f00-aea9-11eb-8fc4-b7de1f01acbc.png)

Instead of the normal application name (`Document Archiver`), the main-class name shows up (`com.sophisticatedapps.archiving.documentarchiver.App`).

I had the same problem and fortunately I found a solution:
[StackOwerflow - javafx application name on gnome](https://stackoverflow.com/a/54467323)

So basically you have to invoke `com.sun.glass.ui.Application.GetApplication().setName("Document Archiver")`.
The problem is that you can't call it in the `Application.start()` method as it's too late, so you have to use a preloader.

Best regards

**Comment by stephansann** [comment]
Hi @Dansoftowner 

Thanks a lot for bringing that up. I will have a look at it.

Best regards
Stephan
